### DISCUSSION TITLE
На скине лвз не хватает деталей

### DISCUSSION THREAD

**Discussion by cyber-dream** [metrostroi-repo/MetrostroiAddon#209] [General]

**Comment by MrSandManFM** [comment]
- [ ] Клепки на бампере

![image](https://user-images.githubusercontent.com/4011223/115706733-a4209800-a376-11eb-88a4-8c6f5ac45f43.png)

- [ ] Надписи на морде (номер вагона, приписка к ТЧ)

- [ ] Шов кузова с базой

![image](https://user-images.githubusercontent.com/4011223/115706835-bd294900-a376-11eb-8f27-b9c197a09172.png)
